R/vector_control.R

```diff
@@ -155,6 +155,44 @@ throw_away_nets <- function(variables) {
   }
 }
 
+
+
+
+
+#' @title Distribute larval source habitat management
+#' @description simulates larval source management by reducing recruitment to adult female mosquitoes 
+#' from `set_habitat_management` and correlation parameters from
+#' `get_correlation_parameters`
+#'
+#' @param variables list of variables in the model
+#' @param larvi_min is the fractional reduction in adult females joining susceptible population 
+#' @param habitat_management_timesteps the time when this is switched on 
+#' @param parameters the model parameters
```

Comment:
I think 'parameters' is actually the first argument in the function

Reply:
thanks have re-ordered and described these more clearly